### PR TITLE
Navbar and Homepage

### PR DESCRIPTION
**Pull request by thibui0908**

### CODE REVIEW

File: src/Components/Navigation/MainNavigation.js [hunk 1]
```diff
@@ -3,34 +3,33 @@ import { Link } from 'react-router-dom';
 
 import MainHeader from './MainHeader';
 import NavLinks from './NavLinks';
-import SideDrawer from './SideDrawer';
 import Backdrop from '../UIElements/Backdrop';
 import './MainNavigation.css';
 
 const MainNavigation = props => {
-  const [drawerIsOpen, setDrawerIsOpen] = useState(false);
+  // const [drawerIsOpen, setDrawerIsOpen] = useState(false);
```
Comment (IsitaB) [inline src/Components/Navigation/MainNavigation.js]: get rid of commented code

File: src/Components/Navigation/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  z-index: 4;
+  align-items: center;
+  font-weight: bold;
+  padding: 15px 0;
+}
+
+.navbrand {
+  padding: 15px;
+}
+.nav-item a {
```
Comment (IsitaB) [inline src/Components/Navigation/NavBar.css:16]: blank line

File: src/Components/Navigation/NavBar.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  color: white !important;
+  fill: white;
+  text-decoration: none;
+  padding: 0 30px;
+  float: right;
+}
+
+.nav-item {
+  display: inline;
+}
```
Comment (IsitaB) [inline src/Components/Navigation/NavBar.css]: blank line at the end

File: src/Components/Navigation/NavLinks.js [hunk 4]
```diff
@@ -11,10 +11,15 @@ const NavLinks = props => {
     <ul className="nav-links">
```
Comment (IsitaB) [inline src/Components/Navigation/NavLinks.js]: Delete unecessary files

File: src/Pages/About.js [hunk 5]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { Container } from 'reactstrap';
-
+import NavBar from '../Components/Navigation/NavBar';
 const About = () => {
   return (
-    <Container>
+    <Container style={{ color: "black" }}>
```
Comment (IsitaB) [inline src/Pages/About.js]: delete inline styling

File: src/Pages/About.js [hunk 6]
```diff
@@ -1,11 +1,12 @@
...
       <h1>About</h1>
       <p>Need club</p>
+      <img src="https://i.insider.com/57ec29fa077dccf2018b8c62?width=1100&format=jpeg&auto=webp" />
```
Comment (IsitaB) [inline src/Pages/About.js]: Let's get rid of pepe too D:

File: src/Pages/Home.css [hunk 7]
```diff
@@ -0,0 +1,4 @@
+img {
+  width: 100%;
+  height: auto;
+}
```
Comment (IsitaB) [inline src/Pages/Home.css]: blank line at the end

File: src/Pages/Home.js [hunk 8]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import { Container } from 'reactstrap';
+import './Home.css';
 
 const Home = () => {
   return (
-    <Container>
+    <Container fluid>
+      <img src="https://www.solidbackgrounds.com/images/website/950x534/950x534-little-green-leaves-free-website-background-image.jpg"></img>
```
Comment (IsitaB) [inline src/Pages/Home.js]: this probably doesnt get past lint

File: src/Routing.js [hunk 9]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
-
 import Home from './Pages/Home';
 import About from './Pages/About';
+import NavBar from './Components/Navigation/NavBar';
+import { Navbar } from 'reactstrap';
```
Comment (IsitaB) [inline src/Routing.js]: delete

File: src/Routing.js [hunk 10]
```diff
@@ -12,20 +13,13 @@ const Routing = () => {
...
-                path={path}
-                component={Component}
-              />
-            );
-          }
-        )}
-        <Redirect  to='/'/>
+        {signedOutRoutes.map(({ Component, path }, index) => {
+          return  (
+            <Route key = { index } exact path = { path } component = { Component } />);
```
Comment (IsitaB) [inline src/Routing.js]: put the attributes on multiples lines please. don't put empty spaces next to these; this is a dynamic rendering thing; the imports at the top using curly braces do need empty spaces

**Review by IsitaB — CHANGES_REQUESTED** [review]
Nice work! Could you figure out a way to ensure that the navbar stays fixed at the top of the viewport?

File: src/Components/Navigation/NavBar.css (hunk 2 above)
Comment (NEED-Development) [reply src/Components/Navigation/NavBar.css:16]: add blank line

**Review by IsitaB — APPROVED** [review]

**Review by NEED-Development — APPROVED** [review]

**Pull request merged**